### PR TITLE
Obsoleting liquidity marketplace spec until we figure out the rest

### PR DESCRIPTION
**Pull request by aljazceru**
obsolete

### CODE REVIEW

File: liquidity-marketplace-spec.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Liquidity marketplace spec
 
+# !!! OBSOLETE - will be reviewd after other proposals are finished !!!
```
Comment (SeverinAlexB) [inline liquidity-marketplace-spec.md:3]:
```suggestion
**This liquidity marketplace specification is OBSOLETE**. The quality of this specification is not sufficient and there is currently no interest to improve it. Therefore, we have made it obsolete. We discourage any usage.
```

**Comment by SeverinAlexB** [comment]
Background: We like to mark this spec obsolete to prevent random outside people from implementing it as we consider the spec insufficient. If there is any interest to work on it in the future, we can always pick it up again.

**Comment by aljazceru** [comment]
> Rather than just adding a note, can we just remove the file for the time being? I think if we pick it up again this should be an entirely new PR, also including a new spec numbering and possibly rather as an extension.
> 
> Having it entirely removed would also safely allow us to renumber LSPS3+ in a reasonable way, without risking confusion in the future.

makes sense

**Comment by ZmnSCPxj-jr** [comment]
Will rename LSPS3+ once this gets merged.

**Pull request merged**